xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -1578,6 +1578,47 @@ void CActiveAE::ChangeResamplers()
   }
 }
 
+static CAEChannelInfo GetInternalChannelLayout(AEStdChLayout stdLayout)
```

Comment:
that method seems rather useless to me. it converts from AE formats to AV and back to AE. Please drop and assign the stdChannelLayout directly